column_operations/rasgo_datetrunc/rasgo_datetrunc.yaml

```diff
@@ -0,0 +1,15 @@
+--
+
+name: rasgo_datetrunc
+source_code: rasgo_datetrunc.sql
+description: Truncates a date to the datepart you specify. For example, if you truncate the date '10-31-2022' to the 'month', you would get '10-1-2022'.
+arguments:
+  date_column:
+    type: string, list
```

Comment:
Also should we make a `column`/`columns` type? This way an auto-populated select can be used in the AI if we know the source. 

May be tricky to work for chaining though since we'll need to run the previous query to get the values.

Reply:
> May be tricky to work for chaining though since we'll need to run the previous query to get the values.

Once the transforms are defined within the context of a collection/block, the chaining experience should be a bit different.  I agree that determining the output columns is an important problem to solve; we should not need to run a transform to get the names of its output columns in most cases.